xclim/indicators/atmos/_temperature.py

```diff
@@ -56,54 +54,39 @@
 # E.g. http://vocab.nerc.ac.uk/collection/P07/current/BHMHISG2/
 
 
-class Tas(Indicator):
+class Tas(Daily):
     """Class for univariate indices using mean daily temperature as the input."""
 
-    def cfprobe(self, tas):
-        cfchecks.check_valid(tas, "cell_methods", "**time: mean within days*")
+    @staticmethod
```

Comment:
Is it normal that Tas.cfcheck() is static while, for example, Tasmin.cfcheck() Tasmax.cfcheck() are not static?

Reply:
No. Fixed.